internal/zinc-persist/src/main/scala/sbt/internal/inc/converters/ProtobufReaders.scala

```diff
@@ -0,0 +1,643 @@
+/*
+ * Zinc - The incremental compiler for Scala.
+ * Copyright 2011 - 2017, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * This software is released under the terms written in LICENSE.
+ */
+
+package sbt.internal.inc.converters
+
+import java.io.File
+
+import sbt.inc.ReadMapper
+import sbt.internal.inc.Relations.ClassDependencies
+import sbt.internal.inc._
+import sbt.util.InterfaceUtil
+import xsbti.{ Position, Problem, Severity, T2, UseScope }
+import xsbti.compile.{ CompileOrder, FileHash, MiniOptions, MiniSetup, Output, OutputGroup }
+import xsbti.compile.analysis.{ Compilation, SourceInfo, Stamp }
+import sbt.internal.inc.converters.ProtobufDefaults.Feedback.StringToException
+import sbt.internal.inc.converters.ProtobufDefaults.Feedback.{ Readers => ReadersFeedback }
+import sbt.internal.inc.converters.ProtobufDefaults.{ Classes, ReadersConstants }
+import sbt.internal.util.Relation
+import xsbti.api._
+
+final class ProtobufReaders(mapper: ReadMapper) {
+  def fromPathString(path: String): File = {
```

Comment:
Why? It converts a path (represented by a string) to a file. The name only describes `from` not `to`. It's the convention all over the place.